examples/custom_search_method/core_search_runner/run_experiment.py

```diff
@@ -0,0 +1,67 @@
+"""
+This script runs a custom SearchMethod with CoreSearchRunner on a Determined cluster.
+
+CoreSearchRunner is responsible for:
+ -> executing the custom SearchMethod as a single trial experiment on the Determined cluster,
+ -> creating a multi-trial experiment on the Determined cluster,
+ -> handling communication between the multi-trial experiment and the custom SearchMethod,
+ -> enabling fault tolerance for SearchMethods that implement save_method_state() and
+    load_method_state().
+
+CoreSearchRunner receives SearcherEvents from the multi-trial experiment, and passes
+the events to your custom SearchMethod, which, in turn, produces a list of Operations.
+Next, CoreSearchRunner sends the operations to the multi-trial experiment for execution.
+"""
+
+import logging
+import determined as det
+from asha import ASHASearchMethod
+from determined.searcher.core_search_runner import CoreSearchRunner
+
+if __name__ == "__main__":
+    logging.basicConfig(level=logging.INFO, format=det.LOG_FORMAT)
+
+
+    ########################################################################
```

Comment:
nice comments!  in terms of style, i usually see triple quotes for multiline comments in python.  also in this specific case, since it's a tutorial example, let's run black formater on the python files.